pkgs/tools/security/sn0int/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, rustPlatform,
+  libsodium, libseccomp, sqlite, pkgconfig }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sn0int";
+  version = "0.18.2";
+
+  src = fetchFromGitHub {
+    owner = "kpcyrd";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0b21b0ryq03zrhqailg2iajirn30l358aj3k44lfnravr4h9zwkj";
+  };
+
+  cargoSha256 = "1pvn0sc325b5fh29m2l6cack4qfssa4lp3zhyb1qzkb3fmw3lgcy";
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [ libsodium libseccomp sqlite ];
+
+  # tries to read /etc/resolv.conf, hence fails in sandbox
+  doCheck = false;
```

Comment:
this seems weird https://github.com/kpcyrd/sn0int/blob/b4fbca4e0df80f9d72a514fb3f7d55015200a2f8/src/sandbox/mod.rs#L65 I don't know a lick of rust but `#[cfg(target_os = "openbsd")]` looks like this statement would only happen on openbsd

Reply:
One of the dependencies ([https://github.com/kpcyrd/chrootable-https/blob/e8e8e68d0e28ccef78a365af30f53d191b48dd2c/src/dns/mod.rs](https://github.com/kpcyrd/chrootable-https/blob/e8e8e68d0e28ccef78a365af30f53d191b48dd2c/src/dns/mod.rs)) tries to read `/etc/resolv.conf` in the `checkPhase`, hencd fails in sandbox of `nix`.